modules/menu/menu.c

```diff
@@ -403,6 +403,34 @@ static int cmd_ua_next(struct re_printf *pf, void *unused)
 }
 
 
+static int cmd_ua_find(struct re_printf *pf, void *arg)
+{
+	const struct cmd_arg *carg = arg;
+	struct ua *ua = NULL;
+
+	if (str_isset(carg->prm)) {
+
+		mbuf_rewind(menu.dialbuf);
+		(void)mbuf_write_str(menu.dialbuf, carg->prm);
```

Comment:
please also remove these two lines, as they are not relevant anymore

Reply:
done